cypress/integration/workspaces/SelectWorkspaces.spec.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+import { Login, Workspaces, Scenarios } from '../../commons/actions';
+import { WORKSPACE_EXAMPLE } from '../../fixtures/stubbing/default';
+import { stub } from '../../commons/services/stubbing';
+import { setup } from '../../commons/utils/setup';
+
+describe('Check workspaces features', () => {
+  before(() => {
+    setup.initCypressAndStubbing();
+    stub.start();
+  });
+
+  after(() => {
+    stub.stop();
+  });
+
+  it('Can select one workspace then return to workspace selector by home button', () => {
+    Login.loginWihtoutWorkspace();
+    Workspaces.getWorkspacesView();
```

Comment:
You're getting an element but do nothing with it, maybe you should add a simple exist/visible check ?

Reply:
It's a mess. I should have test if it is visible.